jobs/package-dockertested/Jenkinsfile

```diff
@@ -21,23 +21,33 @@ node('openshift-build-1') {
 	]])
 	// https://issues.jenkins-ci.org/browse/JENKINS-33511
 	env.WORKSPACE = pwd()
+	def packages = ['docker', 'container-selinux', 'container-storage-setup', 'skopeo', 'atomic', 'python-pytoml']
+	def installed_packages = []
+
 	stage ('Check to see if we need to run') {
-	    next_docker = latestVersion('docker', 'rhel7next*')
-	    next_cselinux = latestVersion('container-selinux', 'rhel7next*')
-	    next_cstorage = latestVersion('container-storage-setup', 'rhel7next*')
-	    next_skopeo = latestVersion('skopeo', 'rhel7next*')
-	    next_atomic = latestVersion('atomic', 'rhel7next*')
-	    echo "rhel7next: docker-${next_docker} container-selinux-${next_cselinux} container-storage-setup-${next_cstorage} skopeo-${next_skopeo} atomic-${next_atomic}"
-	    test_docker = latestVersion('docker', 'dockertested')
-	    test_cselinux = latestVersion('container-selinux', 'dockertested')
-	    test_cstorage = latestVersion('container-storage-setup', 'dockertested')
-	    test_skopeo = latestVersion('skopeo', 'dockertested')
-	    test_atomic = latestVersion('atomic', 'dockertested')
-	    echo "dockertested: docker-${test_docker} container-selinux-${test_cselinux} container-storage-setup-${test_cstorage} skopeo-${test_skopeo} atomic-${test_atomic}"
-	    if ( next_docker == test_docker && next_cselinux == test_cselinux && next_cstorage == test_cstorage && next_skopeo == test_skopeo && next_atomic == test_atomic) {
+		def versions = [:]
+		for(package in packages) {
+			def package_versions = [:]
+			package_versions['next'] = latestVersion(package, 'rhel7next*')
+			package_versions['current'] = latestVersion(package, 'dockertested')
+			versions[package] = package_versions
+		}
+
+		def sync_necessary = false
+		def table = new StringBuffer()
+		table << 'Package'.center(30) << 'Current'.center(30) << 'RHEL 7 Next'.center(30) << '\n'
```

Comment:
.center, .padRight are also restricted. I can add them if really necessary, but can't we get by with `\t` tabbing for imperfect but legible output?